docs/hand_wire.md

```diff
@@ -100,30 +99,115 @@ Things act as they should! Which will get us the following data:
+* Adequate ventilation/a fan
+* Wire cutters/snippers
 
-## Preparing the Diodes
+Optional but useful:
 
-It's a little easier to solder the diodes in place if you bend them at a 90º angle immediately after the black line - this will help to make sure you put them on the right way (direction matters), and in the correct position. The diodes will look like this when bent (with longer leads):
+* Wire strippers/a sharp knife
+* Tweezers and/or small needle nose pliers
+* Soldering station/Helping hands
 
-```
-       ┌─────┬─┐
-    ───┤     │ ├─┐
-       └─────┴─┘ │
-                 │
-```
+## Starting the build
+
+There are many ways to hand wire a PCB matrix, this guide will describe the fundamentals as well as some recommended ways to go about it.
+
+As we are dealing with hand wiring, it is assumed that you already have a plate.  If you are planning a completely custom layout, tools such as [ai03 Plate Generator](https://kbplate.ai03.me/) and [Swillkb Plate & Case Builder](http://builder.swillkb.com/) can help when designing one.
+
+Start by installing the switches and stabilisers in the plate. Depending on the thickness and material this may also involve hot gluing it in place.
+
+## Planning the matrix
+
+What you want to achieve is one leg from each switch being attached to the corresponding switches next to it (rows) and the other leg being attached to the switches above and below it (columns) and a diode to one of the legs, mosy commonly this will be the leg attached to the rows, and the diode will face away from it (Column to Row) i.e. with the wire furthest from the black line on the diode connected to the switch (as current will only travel in one direction through a diode)
+
+It is fairly simple to plan for an ortholinear keyboard (like a planck) 
+
+![Example planck matrix](https://i.imgur.com/FRShcLD.png)
+Image from [RoastPotatoes' "How to hand wire a Planck"](https://blog.roastpotatoes.co/guide/2015/11/04/how-to-handwire-a-planck/)
+
+But the larger and more complicated your keyboard, the more complex the matrix.  [Keyboard Firmware Builder](https://kbfirmware.com/) can help you plan your matrix layout (shown here with a basic fullsize ISO keyboard imported from [Keyboard Layout Editor](http://www.keyboard-layout-editor.com)
+
+![Example ISO matrix](https://i.imgur.com/UlJ4ZDP.png)
+
+Also bear in mind that the number of rows plus the number of columns can not exceed the number of I/O pins on your controller.  So the fullsize matrix shown above would be possible on a Proton C or Teensy++, but not on a regular Teensy or Pro Micro
+
+#### Common Microcontroller Boards
+
+| Board         | Controller    | # I/O  | Pinout |
+| :------------ |:-------------:| ------:| ------ |
+| Pro Micro     | ATmega32u4    |     20 | [link](https://learn.sparkfun.com/tutorials/pro-micro--fio-v3-hookup-guide/hardware-overview-pro-micro)       |
+| Teensy 2.0    | ATmega32u4    |     25 | [link](https://www.pjrc.com/teensy/pinout.html)       |
```

Comment:
Maybe add the Elite-c here?

Reply:
Elite C uses the same pinout as the Pro Micro, with some extra pins. And there is no official page for it.

Reply:
Added a note about the Elite C and made a reference to split keyboards with a link to the relevant page